rust/optify/src/provider/watcher.rs

```diff
@@ -124,23 +124,23 @@ impl OptionsWatcher {
                             Ok(mut provider) => {
                                 *provider = new_provider;
                                 *last_modified.lock().unwrap() = std::time::SystemTime::now();
-                                println!("\x1b[32m[optify] Successfully rebuilt the OptionsProvider.\x1b[0m");
+                                eprintln!("\x1b[32m[optify] Successfully rebuilt the OptionsProvider.\x1b[0m");
```

Comment:
Those one too. Why eprintln?

Reply:
My general rule is all log messages should go to stderr.  Otherwise it's impossible to write a tool with parseable output.  For example if you're printing some JSON to stdout, possibly piping it to `jq` or something, and right in the middle of it optify prints something, it will be invalid JSON (or worse, valid but wrong)

Reply:
Thought so. Alright.